pkg/controller/realmkeys/save.go

```diff
@@ -61,8 +61,11 @@ func (c *Controller) HandleSave() http.Handler {
 		}
 
 		// Update settings.
-		currentRealm.CertificateIssuer = form.Issuer
-		currentRealm.CertificateAudience = form.Audience
+		if !currentRealm.UseRealmCertificateKey {
```

Comment:
Should this be an error instead?

Reply:
I went back and forth on this - I went w/ the easier thing since this is unlikely to happen to a human not trying to get around this.